theme_aviato/views/snippets/s_picture.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <template id="s_picture" inherit_id="website.s_picture">
+        <xpath expr="//section" position="attributes">
+            <attribute name="class" add="pt72 overflow-hidden o_cc3" remove="pt48 o_cc2" separator=" "/>
+            <attribute name="data-oe-shape-data">{"shape":"web_editor/Bold/01","flip":[]}</attribute>
+        </xpath>
+
+        <xpath expr="//section/div" position="before">
+            <div class="o_we_shape o_web_editor_Bold_01 bg-o-color-3"/>
+        </xpath>
```

Comment:
This is good (adding o_cc3 as preset, forcing the bold shape to use the same bg color as that preset so that the text renders well over the shape, then add a bg color on the shape itself) :+1:

However, it may be not perfect in the new system: when dropping that snippet, the user has to understand that to control the background color of the snippet, it has to first remove the shape / reset it background transparency :grimacing: Maybe it is best to leave the shape background transparent when we can: now you can add that `bg-o-color-3` class on the snippet itself **alongside the o_cc3 class**. That will have the same visual aspect of your snippet here but the editor panel status and further edition will make more sense IMO.